dev/com.ibm.ws.transport.http/src/io/openliberty/http/options/HttpOption.java

```diff
@@ -38,7 +37,7 @@ public enum HttpOption implements EndpointOption {
     THROW_IOE_FOR_INBOUND_CONNECTIONS("ThrowIOEForInboundConnections", null, Boolean.class, ConfigType.HTTP),
     DECOMPRESSION_RATIO_LIMIT("decompressionRatioLimit", 200, Integer.class, ConfigType.HTTP),
     DECOMPRESSION_TOLERANCE("decompressionTolerance", 3, Integer.class, ConfigType.HTTP),
-    HTTP2_CONNECTION_IDLE_TIMEOUT("http2ConnectionIdleTimeout", "0", String.class, ConfigType.HTTP2),
+    HTTP2_CONNECTION_IDLE_TIMEOUT("http2ConnectionIdleTimeout", 0, Integer.class, ConfigType.HTTP2),
```

Comment:
Why are we changing this? Docs say "A period of time with second precision", can we still set something like 1h30s?